fmt-readme

```diff
@@ -0,0 +1,2 @@
+remark README.md  > README.md.2 && mv README.md.2 README.md
```

Comment:
Useful for the README but the main part of this repo is for `docs/index.md`, have you tried running it over it?